packages/@aws-cdk/aws-iotevents/README.md

```diff
@@ -40,15 +40,30 @@ Import it into your code:
 import * as iotevents from '@aws-cdk/aws-iotevents';
 ```
 
-## `Input`
+## `DetectorModel`
 
-Add an AWS IoT Events input to your stack:
+The following example creates an AWS IoT Events detector dodel to your stack.
+The detector model need a reference to at least one AWS IoT Events input.
+AWS IoT Events input enable that the detector can get MQTT payload values from IoT Core rules.
 
 ```ts
 import * as iotevents from '@aws-cdk/aws-iotevents';
 
-new iotevents.Input(this, 'MyInput', {
-  inputName: 'my_input',
+const input = new iotevents.Input(this, 'MyInput', {
+  inputName: 'my_input', // optional
   attributeJsonPaths: ['payload.temperature'],
 });
+
+const onlineState = new iotevents.State({
+  stateName: 'online',
+  onEnterEvents: [{
+    eventName: 'test-event',
+    condition: `currentInput("${input.inputName}")`,
```

Comment:
What's this `currentInput` thing?

Reply:
`currentInput` is [the function defined by IoT Events](https://docs.aws.amazon.com/iotevents/latest/developerguide/iotevents-expressions.html#:~:text=currentInput(%22input%22)).
This function is one of most commonly used functions. So I'm thinking of making this feature part of the Input functionality as following:

```ts
class Input {
  public getCurrentInputCondition() {
    return `currentInput("${this.inputName}")`;
  }
}
```

Reply:
Yes, I like that.